interfaces/features/shared/Control.d.ts

```diff
@@ -36,14 +36,3 @@ interface Modifier {
     target: Feature["id"];
     handler: Controllers | { name: "SomeOtherTypeOFModififer" };
 }
-
```

Comment:
Statements are not allowed in .d.ts files, and this didn't look like it was doing anything.